WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingView.swift

```diff
@@ -5,53 +5,69 @@ struct StoreOnboardingView: View {
-            StoreSetupProgressView(isExpanded: viewModel.isExpanded,
-                                   totalNumberOfTasks: viewModel.taskViewModels.count,
-                                   numberOfTasksCompleted: viewModel.numberOfTasksCompleted)
-
-            // Task list
-            VStack(alignment: .leading, spacing: Layout.verticalSpacingBetweenTasks) {
-                ForEach(Array(viewModel.taskViewModels.enumerated()), id: \.offset) { index, taskViewModel in
-                    let isLastTask = index == viewModel.taskViewModels.count - 1
-
-                    StoreOnboardingTaskView(viewModel: taskViewModel,
-                                            showDivider: !isLastTask) { task in
-                        taskTapped(task)
+        VStack {
+            Color(.systemColor(.systemGray6))
+                .frame(height: 16)
+                .renderedIf(!viewModel.isExpanded)
+
+            let verticalSpacing = viewModel.isExpanded ? Layout.VerticalSpacing.expandedMode : Layout.VerticalSpacing.collapsedMode
+            VStack(alignment: viewModel.isExpanded ? .center : .leading, spacing: verticalSpacing) {
+                DismissButton(action: dismissAction)
+                    .renderedIf(viewModel.isExpanded)
+
+                // Progress view
+                StoreSetupProgressView(isExpanded: viewModel.isExpanded,
+                                       totalNumberOfTasks: viewModel.taskViewModels.count,
+                                       numberOfTasksCompleted: viewModel.numberOfTasksCompleted)
+
+                // Task list
+                VStack(alignment: .leading, spacing: Layout.verticalSpacingBetweenTasks) {
+                    ForEach(Array(viewModel.tasksForDisplay.enumerated()), id: \.offset) { index, taskViewModel in
+                        let isLastTask = index == viewModel.tasksForDisplay.count - 1
+
+                        StoreOnboardingTaskView(viewModel: taskViewModel,
+                                                showDivider: !isLastTask) { task in
+                            taskTapped(task)
+                        }
                     }
                 }
-            }
 
-            // View all button
-            Button {
-                viewAllTapped?()
-            } label: {
-                Text(String(format: Localization.viewAll, viewModel.taskViewModels.count))
-                    .fontWeight(.semibold)
-                    .foregroundColor(.init(uiColor: .accent))
-                    .subheadlineStyle()
+                // View all button
+                ViewAllButton(action: viewAllTapped, text: String(format: Localization.viewAll, viewModel.taskViewModels.count))
+                    .renderedIf(!viewModel.isExpanded)
+
+                Spacer()
+                    .renderedIf(viewModel.isExpanded)
             }
-            .renderedIf(!viewModel.isExpanded)
+            .padding(insets: Layout.insets)
+
+            Color(.systemColor(.systemGray6))
```

Comment:
Same comment about using `.systemColor`.

Reply:
Done in 5787ab7736b5036b2b42aba78707031cbc41cc12